src/libraries/Common/src/Interop/Browser/Interop.Runtime.Api.cs

```diff
@@ -0,0 +1,386 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
.Api is not the best name (what about .VM.cs or .Bridge.cs`). Could you also please add a link to source location where this is called from?

Reply:
These are not called by anything right now as far as I know.  Not until the https://github.com/dotnet/runtime/pull/35566 is complete.  The file will be `src/mono/wasm/runtime/corebindings.c `

Reply:
Changed to `Interop.Runtime.Bridge.cs`